### PR TITLE
[BUG] Fix bug of vis-cam

### PR DESCRIPTION
**Pull request by Ezra-Yu**
## Motivation

Fix bug in #654 , `support gram-cam==1.3.7`.

## Modification

Fix bug in #654 

## Checklist

**Before PR**:

- [x] Pre-commit or other linting tools are used to fix the potential lint issues.
- [x] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: tools/visualizations/vis_cam.py [hunk 1]
```diff
@@ -327,10 +328,27 @@ def main():
...
+        try:
+            from pytorch_grad_cam.utils.model_targets import (
+                ClassifierOutputTarget)
+            target_category = [
+                ClassifierOutputTarget(category)
+                for category in args.target_category
+            ]
+        except Exception as e:
+            import warnings
+            warnings(f'You can using pytorch_grad_cam==1.3.6 : {e}')
```
Comment (mzr1996) [inline tools/visualizations/vis_cam.py:345]: unnecessary warning.

**Review by mzr1996 — CHANGES_REQUESTED** [review]
Better to develop based on #656

File: tools/visualizations/vis_cam.py (hunk 1 above)
Comment (Ezra-Yu) [reply tools/visualizations/vis_cam.py]: done.

**Comment by Ezra-Yu** [comment]
close due to #656

**Pull request closed without merging**